### PR TITLE
Add external postgres documentation

### LINKED ISSUE #7189: Document external postgresql database requirements
When using Harbor with an external database, how should I prep my postgresql instance? Should I create any databases/tables? Or is this handled by Harbor on startup?

From my own experimentation, it seems like one needs to create the following:
```
-- Create required databases
CREATE DATABASE notaryserver;
CREATE DATABASE notarysigner;
CREATE DATABASE registry ENCODING 'UTF8';
CREATE DATABASE clair;

-- Create harbor user
-- The helm chart limits us to a single user for all databases
CREATE USER harbor;
ALTER USER harbor WITH ENCRYPTED PASSWORD 'change-this-password';

-- Grant the user access to the DBs
GRANT ALL PRIVILEGES ON DATABASE notaryserver TO harbor;
GRANT ALL PRIVILEGES ON DATABASE notarysigner TO harbor;
GRANT ALL PRIVILEGES ON DATABASE registry TO harbor;
GRANT ALL PRIVILEGES ON DATABASE clair to clair;
```

### PR DESCRIPTION
**Pull request by alexbrand**
Fixes #7189

### CODE REVIEW

**Comment by alexbrand** [comment]
Thanks @nlowe. I am actually not sure how to change the conn string through the installer. It seems like the installer assumes postgres is always going to be in the docker-compose file?

**Comment by nlowe** [comment]
In the `harbor.cfg` for my `docker-compose` installation of harbor, I see the following properties:

```python
#######Harbor DB configuration section#######

#The address of the Harbor database. Only need to change when using external db.
db_host = postgresql

#The password for the root user of Harbor DB. Change this before any production use.
db_password = ***

#The port of Harbor database host
db_port = 5432

#The user name of Harbor database
db_user = postgres

##### End of Harbor DB configuration#######
```

I'm guessing you would just need to change the `db_host` to point to your external database. However, I don't see this in our [Installation Guide](https://github.com/goharbor/harbor/blob/master/docs/installation_guide.md). @steven-zou Am I correct on this, using `db_host` to point to an external database?

**Comment by mmpei** [comment]
you are right for creating these databases when using external database. i also don't find any related specification when i occured this problem. maybe you can add more description for how to set these DB paramters in harbor config file.

File: docs/external-postgres.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+-- Create harbor user
+-- The helm chart limits us to a single user for all databases
+CREATE USER harbor;
+ALTER USER harbor WITH ENCRYPTED PASSWORD 'change-this-password';
+
+-- Grant the user access to the DBs
+GRANT ALL PRIVILEGES ON DATABASE notaryserver TO harbor;
+GRANT ALL PRIVILEGES ON DATABASE notarysigner TO harbor;
+GRANT ALL PRIVILEGES ON DATABASE registry TO harbor;
+GRANT ALL PRIVILEGES ON DATABASE clair to clair;
```
Comment (leosunmo) [inline docs/external-postgres.md]: Shouldn't this be 
```suggestion
GRANT ALL PRIVILEGES ON DATABASE clair TO harbor;
```

**Comment by reasonerjt** [comment]
We have modified the installation process, and updated the doc in #7761
Closing this PR.
@alexbrand thanks for the contribution

**Pull request closed without merging**